README.md

```diff
@@ -1,4 +1,4 @@
-# Booking Manager Connector
+# === Booking Manager Connector ===
```

Comment:
The equal signs will be rendered verbatim on GitHub. Is that intended?

Reply:
Yes it should really look like "=== ... ===" - encapsulated in "=" signs.